client/angular/controllers/MainController.js

```diff
@@ -193,6 +208,9 @@
             } else {
                 $scope.displayColumns = sortedColumns;
             }
+            $scope.onscreenTweets = (backfilledColumns.reduce(function(prevColumn, curColumn) {
```

Comment:
Actually scratch that, I see how it's being used on the mobile view. Should be fine